packages/redux-devtools-app/jest.config.js

```diff
@@ -3,6 +3,6 @@ module.exports = {
   setupFilesAfterEnv: ['<rootDir>/test/setup.ts'],
   testEnvironment: 'jsdom',
   moduleNameMapper: {
-    '\\.css$': '<rootDir>/test/__mocks__/styleMock.ts',
+    '\\.css$': '<rootDir>/test/__mocks__/styleMock.js',
```

Comment:
Was it cause problems for this to be `ts`?

Reply:
I do not recall exactly, luckily I wrote a detailed commited message about it.

```ts
chore(@redux-devtools/app): fix lint error

Error message:

0:0  error  Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
The file does not match your project config: test/__mocks__/styleMock.ts.
The file must be included in at least one of the projects provided
```

Reply:
Nice commit message! I wonder if those was fixed by https://github.com/reduxjs/redux-devtools/pull/867.